manifests/scenario/simp.pp

```diff
@@ -23,6 +23,7 @@
   # Ensuring reasonably sane defaults
   include 'at'
   include 'cron'
+  include 'fips'
```

Comment:
@lnemsick-simp No, this should not go in `simp-lite` since that one is targeted toward non-scary stuff.